src/core/tooltips/rfc.md

```diff
@@ -0,0 +1,86 @@
+## Tooltip Service
+
+This service allows us to show tooltips in different places of our application.
+Underhood it uses `@konturio/ui-kit` tooltip component for show tooltip in theme style
+
+Tooltip:
+
+- Shows by hover-in and hides by hover-out events (after delay)
+- Not interactive
+- Can contain only text
+
+## How to use
+
+We assume that all cases of using a tooltip can be reduced to two cases
+
+### We have an element around which a tooltip should be shown
+
+In that case you need to import special components and use them
+
+```jsx
+import { Tooltip, TooltipTrigger, TooltipContent } from '~core/tooltip';
+
+function Sidebar() {
+  return (
+    <nav>
+      <Tooltip>
```

Comment:
If we need text-only content, we can use a simpler design

```tsx
<Tooltip content="string-only-content">
  {/* Where to show */}
  <button>Some button with tip</button>
</Tooltip>
```

Reply:
It's better to keep like this.
I can imagine case when you want pass <Trans>  component, or <Markdown> to tootip content.
Also in feature we probably add settings for content and trigger, and it will be cool that we keep it separately